leapp/snactor/__init__.py

```diff
@@ -68,6 +79,17 @@ def cli(args):
 
 
 def main():
+    profile_enabled = os.environ.get('LEAPP_CPROFILE', '0') == '1'
+    if profile_enabled:
+        pr = cProfile.Profile()
+        pr.enable()
     os.environ['LEAPP_HOSTNAME'] = socket.getfqdn()
     load_commands()
     cli.command.execute(version=_('snactor version {}').format(VERSION))
+    if profile_enabled:
+        pr.disable()
+        s = StringIO.StringIO()
+        sortby = 'cumulative'
+        ps = pstats.Stats(pr, stream=s).sort_stats(sortby)
+        ps.print_stats()
+        print(s.getvalue())
```

Comment:
I'd rather like to see this being printed to stderr

@AloisMahdal what's your take on this?